unstructured/partition/common.py

```diff
@@ -407,12 +422,21 @@ def convert_office_doc(
         input_filename,
     ]
     try:
-        process = subprocess.Popen(
-            command,
-            stdout=subprocess.PIPE,
-            stderr=subprocess.PIPE,
-        )
-        output, error = process.communicate()
+        # only one soffice process can be ran
+        wait_time = 0
+        sleep_time = 0.1
```

Comment:
I wonder if a bit of randomness in the sleep time might not be a good idea.

There could be more than one DOC file stacked up and some randomness would work against possible "resonant" behavior where both waiting threads time out at the same time over and over.

Maybe:
```python
sleep_time = 0.095 + random.uniform(0.0, 0.01)
```

One observation from the `partition_doc()` tests is that the file conversion takes quite a bit longer than actually partitioning the result. I expect that would have a tendency to stack up DOC files when they arrived in clusters.

Reply:
sorry didn't see that earlier; this is a good idea; will do that for as a followup